app/src/main/java/com/fruitscale/ananas/login/SessionManager.kt

```diff
@@ -0,0 +1,92 @@
+package com.fruitscale.ananas.login
+
+import android.content.Context
+import com.fruitscale.ananas.store.LoginStorage
+import org.matrix.androidsdk.MXSession
+import org.matrix.androidsdk.rest.model.login.Credentials
+
+/**
+ * Sessions should be managed with the [SessionManager]
+ */
+class SessionManager(val context: Context) {
```

Comment:
This class holds and loads matrix sessions from the app storage. It also refreshes the sessions.